docs/graphql/manual/schema/relationships/architecture.rst

```diff
@@ -0,0 +1,33 @@
+.. meta::
+   :description: Security, performance, schema integrity, and data federation with Hasura remote joins
+   :keywords: hasura, docs, data federation, remote relationship, remote join, remote schema
+
+.. _remote_joins_architecture:
+
+Architecture of remote joins
```

Comment:
Also, we need to be consistent with the naming: either "Remote joins" or "Remote relationships". @rikinsk and me think it should be "Remote relationships" since then the wording is consistent with the rest of the headings 🙂